peachjam/js/locale/sw/translation.json

```diff
@@ -50,6 +50,7 @@
   "Search documents by flynote": "Tafuta nyaraka kwa flynote",
   "Search documents by headnote holding": "Tafuta hati kwa kutumia muhtasari",
   "Search documents by judges": "Tafuta nyaraka kwa kutumia majaji",
+  "Search documents by jurisdiction": "Tafuta nyaraka kwa kutumia mamlaka za kisheria",
   "Search documents by title": "Tafuta hati kwa kutumia kichwa cha habari",
```

Comment:
@longhotsummer I added this translation for swahili. The other locales do not have the translation for this string. i.e french and portugese